crypto/keyring/keyring.go

```diff
@@ -305,14 +305,23 @@ func (ks keystore) ImportPubKey(uid string, armor string) error {
 		return err
 	}
 
-	pubKey, err := legacy.PubKeyFromBytes(pubBytes)
+	pubKey, err := legacy.AminoPubKeyFromBytes(pubBytes)
 	if err != nil {
 		return err
 	}
 
-	_, err = ks.writeOfflineKey(uid, pubKey, hd.PubKeyType(algo))
```

Comment:
Do we agree this was bugged before? If the old keyring had this as `TypeLedger` or `TypeMulti`, the new one will anyways be a `TypeOffline` ...

Reply:
yeah, this was the result of a test we did, we tried also the writeLedger and offline.